raiden-ts/tests/unit/reducers.spec.ts

```diff
@@ -230,6 +233,105 @@ describe('raidenReducer', () => {
     });
   });
 
+  describe('channelWithdrawn', () => {
+    const deposit = bigNumberify(500) as UInt<32>;
+    beforeEach(() => {
+      state = [
+        channelOpened(
+          { id: channelId, settleTimeout, openBlock, isFirstParticipant, txHash },
+          { tokenNetwork, partner },
+        ),
+        channelDeposited(
+          {
+            id: channelId,
+            participant: state.address,
+            totalDeposit: deposit,
+            txHash,
+          },
+          { tokenNetwork, partner },
+        ),
+        channelDeposited(
+          {
+            id: channelId,
+            participant: partner,
+            totalDeposit: deposit,
+            txHash,
+          },
+          { tokenNetwork, partner },
+        ),
+      ].reduce(raidenReducer, state);
+    });
+
+    test('channel not in open state', () => {
```

Comment:
It's a bit hard to follow what the expected result here is

Reply:
I'll try to make it clearer